### DISCUSSION TITLE
What type of unit is the flux measured in?

### DISCUSSION THREAD

**Discussion by LordFurno** [lightkurve/lightkurve#1251] [Q&A] [answered]
I know that the flux is measured in pdscap_flux but how does this compare to other measurements of brightness such as apparent magnitude of stars?

**Comment by warrickball** [comment, marked as answer]
The flux in the light curves (or at least the SPOC pipeline ones) is electrons per second.
```
>>> import lightkurve as lk
>>> lc = lk.search_lightcurve('HD 38529')[0].download()
>>> print(lc.flux.unit)
electron / s
```
Apparent magnitude is a logarithmic measure of flux relative to some zero point (e.g. Vega). There are probably more robust ways of doing this but I usually resort to the value in the light curve metadata (although it isn't *always* reliable).
```
>>> print(lc.meta['TESSMAG'])
5.27250004
```
To plot light curves in magnitudes, I convert the flux to differential magnitudes with something like
```
>>> import numpy as np
>>> delta_mag = -2.5*np.log10(lc.flux.value/np.nanmedian(lc.flux.value))
>>> mag = lc.meta['TESSMAG'] + delta_mag
```
